static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -83,9 +83,10 @@ type Props = {
   location: Location;
   organization: Organization;
   setError: (msg: string | undefined) => void;
-  totalEventCount: string;
   transactionName: string;
   columnTitles?: string[];
+  disablePagination?: boolean; // Workaround for perf issue page
```

Comment:
I think it's straightforward enough just having a disableX feature for some UI that we don't need a comment pertaining to perf issues here, since we're using TS we can always find out usage via references to it.

Reply:
removed